persistence/sql/migrations/20220914000001000000_support_refresh_remember_for.sqlite.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE hydra_oauth2_flow ADD login_extend_session_lifespan BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
The migration timestamp should be later than the last migration which is:

```
20230228000010000001_client_add_skip_consent_column.up
```

Maybe just use the current date:

```
20230313112801000001_support_refresh_remember_for
```